cypress/integration/voter-registration-drive-page.js

```diff
@@ -347,4 +348,30 @@ describe('Voter Registration Drive (OVRD) Page', () => {
       .get('h2')
       .contains(8);
   });
+
+  /** @test */
+  it(' If a group is found, OVRD page displays link to groups campaign page', () => {
```

Comment:
We're testing for certain group ID values in the test above if a group is found, what do you think about renaming this test specifically for testing the link if the group's group type has a website for a voter registration campaign.
```suggestion
  it(' OVRD page displays link to group voter registration campaign website if exists for the group type, ', () => {
```

Let's also check in the test above that the `voter-registration-drive-page-group-campaign-link` isn't rendered.